pkg/e2e/verify/regularuser_webhook.go

```diff
@@ -0,0 +1,43 @@
+package verify
+
+import (
+	"context"
+
+	"github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/openshift/osde2e/pkg/common/alert"
+	"github.com/openshift/osde2e/pkg/common/config"
+	"github.com/openshift/osde2e/pkg/common/helper"
+	"github.com/spf13/viper"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/rest"
+)
+
+var regularuserWebhookTestName string = "[Suite: service-definition] [OSD] regularuser validating webhook"
+
+func init() {
+	alert.RegisterGinkgoAlert(regularuserWebhookTestName, "SD-SREP", "Max Whittingham", "sd-cicd-alerts", "sd-cicd@redhat.com", 4)
```

Comment:
I'm unfamiliar with this, what does it do?

Reply:
It's part of osde2e and allows alerting based on failures of specific tests. If you hop in #sd-cicd-alerts on Slack you can see its behavior.

Reply:
I believe it's what registers the tests with Ginkgo

Reply:
Cool cool :+1:

Reply:
Just to be clear, this isn't a ginkgo construct, it's an OSDe2e construct that generates alerts based on (aggregated) ginkgo failures. :) 

See https://github.com/openshift/osde2e/blob/main/pkg/common/alert/alert.go if you want a rabbit hole. :)